josebasilioo/api-naurto
# Hotfix/serverbase

###What i did
- Implemented the elements endpoint to the serverbase

###How to test
1. Run `npm run nodemon` on `/server`
2. Open insomnia and make the structure equal to `elementsModel` structure (ask me if you need how to make it)
3. Send the method defined on `/Routes/Elements.js` and verify at Atlas database if it works
